tests/scripts/thread-cert/Cert_9_2_13_EnergyScan.py

```diff
@@ -42,23 +44,27 @@ class Cert_9_2_13_EnergyScan(thread_cert.TestCase):
 
     TOPOLOGY = {
         COMMISSIONER: {
+            'name': 'COMMISSIONER',
             'mode': 'rsdn',
             'panid': 0xface,
             'router_selection_jitter': 1,
             'allowlist': [LEADER]
         },
         LEADER: {
+            'name': 'LEADER',
             'mode': 'rsdn',
             'panid': 0xface,
             'allowlist': [COMMISSIONER, ROUTER1]
         },
         ROUTER1: {
+            'name': 'ROUTER',
             'mode': 'rsdn',
             'panid': 0xface,
             'router_selection_jitter': 1,
             'allowlist': [LEADER, ED1]
         },
         ED1: {
+            'name': 'ED',
```

Comment:
In this case, the ED DUT can be MED or SED. How do we verify both role within one case?

Reply:
There is no special packet to verify for MED/SED in the test plan

Reply:
But we need to verify that both MED/SED works in this test case.
The simplest method is to have two almost identical test cases, with different mods for the ED device. But this brings too much redundant code. 

A better way I can think of is to define a base test class and create two subclasses (in two files) one for MED DUT and another for SED DUT. Each subclass define it's own topology with correct modes for SED/MED. 

Thoughts? @FiveDimensions @librasungirl

Reply:
Currently, there are several cases with different DUT scenarios, implement this as a example to refactor the others